Gems/Camera/Code/Source/CameraSystemComponent.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Component/Component.h>
+
+#include <AzFramework/Components/CameraBus.h>
+
+namespace Camera
+{
+    class CameraSystemComponent
+        : public AZ::Component
+        , private CameraSystemRequestBus::Handler
+        , private ActiveCameraRequestBus::Handler
```

Comment:
update/suggestion: Again consider making the request buses public and put them in the public section of the class (notification can totally be private)